src/App.js

```diff
@@ -10,10 +10,23 @@ class App extends React.Component {
     super()
     this.state = {
       movies: [],
+      filteredMovies: [],
       errorMessage: "",
     };
   }
 
+  searchMovies = (event) => {
+    const { value } = event.target
+    const foundMovies = this.state.movies.filter(movie => {
+      if (movie.title.toLowerCase().includes(value.toLowerCase())) {
+        return movie
+      }
+    })
+    this.setState({
+      filteredMovies: foundMovies
+    })
+  }
+
```

Comment:
Great! I like this outcome. We can apply some of what we learned on this to future projects.